### ISSUE TITLE
Hex logo for ocbe organisation 

### ISSUE THREAD

**Issue by chizapoth** [ocbe-uio/ocbe-uio.github.io#3]
Could be as simple as text on a solid color background.

**Comment by wleoncio** [comment]
Yeah, an OCBE logo is something I asked about a few years ago. We might touchg base with Arnoldo to see if there's anthing we could base off. Maybe something merging the UiO and OUS logos?

BTW, this might be a good place to put a link to the official UiO colors, I always like to use them on my logos: https://www.uio.no/om/designmanual/profilelementer/fargepalett/index.html

**Comment by wleoncio** [comment]
The discussion about an OCBE logo will be taken at a higher level. Currently it's unplanned, as OCBE is hosted at a UiO domain, and the university has strict [guidelines for their monolithic logo model](https://www.uio.no/english/about/designmanual/profile-elements/logo/index.html).
